src/config/index.ts

```diff
@@ -23,6 +23,8 @@ const configSchema = {
   TEST_FUND_GRANTS_PROGRAM_WITH_PROPOSAL: parseBoolean({ default: true }),
   FUND_GRANTS_PROGRAM_v1_5_PROPOSAL_ID: parseInteger({ default: null }),
   TEST_FUND_GRANTS_PROGRAM_v1_5_WITH_PROPOSAL: parseBoolean({ default: true }),
+  UPDATE_MERKLE_DISTRIBUTOR_REWARDS_PARAMETERS_PROPOSAL_ID: parseInteger({ default: null }),
```

Comment:
Do you potentially want to update `FORK_BLOCK_NUMBER` to the block number after the latest proposal was passed?

Reply:
Done - updated to 15314711 based on the [execute transaction](https://etherscan.io/tx/0x48540f0a2c0693a1c9c643134e0e4db257a5b9799b14d98742073c8d1eafbbb5)